TempoCore/Source/TempoCore/Public/TempoPlayerController.h

```diff
@@ -1,28 +1,174 @@
-﻿// Copyright Tempo Simulation, LLC. All Rights Reserved
+// Copyright Tempo Simulation, LLC. All Rights Reserved
 
 #pragma once
 
 #include "CoreMinimal.h"
 #include "GameFramework/PlayerController.h"
+#include "Components/SpotLightComponent.h"
 #include "TempoPlayerController.generated.h"
 
+class ASpectatorPawn;
+class UUserWidget; // Forward declaration for UUserWidget class
+
+// A struct to wrap the TArray, making it compatible with TMap UPROPERTYs.
```

Comment:
```suggestion
// A struct to group pawns with a common parent.
```
(don't explain "common knowledge" in comments - Unreal engineers will know you can't have a TArray directly in a TMap)